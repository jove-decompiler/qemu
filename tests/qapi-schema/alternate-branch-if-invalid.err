tests/qapi-schema/alternate-branch-if-invalid.json:2: 'if' condition ' ' makes no sense
