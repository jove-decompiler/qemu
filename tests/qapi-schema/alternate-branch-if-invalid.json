# Cover alternative with invalid 'if'
# FIXME not rejected, would generate '#if  \n'
{ 'alternate': 'Alt',
  'data': { 'branch': { 'type': 'int', 'if': ' ' } } }
