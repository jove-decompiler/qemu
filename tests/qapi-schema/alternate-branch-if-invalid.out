module None
object q_empty
enum QType
    prefix QTYPE
    member none
    member qnull
    member qnum
    member qstring
    member qdict
    member qlist
    member qbool
module alternate-branch-if-invalid.json
alternate Alt
    tag type
    case branch: int
        if [' ']
